docs/services/hpc-console.md

```diff
@@ -0,0 +1,53 @@
+[](){#hpc-console}
+# HPC Console
+
+HCP Console is a web application designed to provide essential functionalities for interacting with [FirecREST API v2 backend][ref-firecrest].
+It provides users with a modern web-based environment to interact with HPC resources such as clusters, job schedulers, and filesystems.
+
+## Features
+
+- A modern web interface built on FirecREST API functionalities.
```

Comment:
I would remove this line, because it's a secondary thing to mention FirecREST as the backend